locales/zh-CN.json

```diff
@@ -0,0 +1,58 @@
+{
+  "projectName": {
+    "message": "请输入项目名称:"
```

Comment:
Please use the full-width colon character `：`
https://github.com/vuejs-translations/docs-zh-cn/wiki/%E7%BF%BB%E8%AF%91%E9%A1%BB%E7%9F%A5#22-%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7